web/Controllers/LandlordController.cs

```diff
@@ -133,4 +132,29 @@ public IActionResult ViewProperties()
         var listOfProperties = databaseResult.Select(PropertyViewModel.FromDbModel).ToList();
         return View("Properties", new PropertiesDashboardViewModel(listOfProperties));
     }
+    
+    [HttpGet]
+    [Route("edit")]
+    public ActionResult EditProfilePage(LandlordProfileModel currentProfileModel)
+    {
+        return View("EditProfilePage", currentProfileModel);
+    }
+
+    [HttpPost]
+    [Route("edit")]
+    public ActionResult EditProfileUpdate([FromForm] LandlordProfileModel createModel)
+    {
+        var user = GetCurrentUser();
+        var isEmailDuplicated = _landlordService.CheckForDuplicateEmail(createModel, user.LandlordId);
+        if (isEmailDuplicated)
+        {
+            _logger.LogWarning("Email already registered {Email}", createModel.Email);
+            ModelState.AddModelError("Email", "Email already registered");
+            return View("EditProfilePage", createModel);
+        }

```

Comment:
I think rather than getting the landlord id from the user, you should supply it in the route or form values, then check the user has permission to edit that landlord. I think that would be slightly clearer to understand and has a couple of advantages:

* Admins can then use the same route to edit landlord's profiles
* If we ever want multiple users per landlord it would just work

Reply:
This was a change I was working on in BNH-45 (Admins edit landlord profile) :) It's why I asked about adding landlord id to the landlord profile model! I can add it in to this ticket if you think that would be a good idea or I can change it in ticket 45, depends what you think.

Reply:
I think after thinking abt it it makes more sense to include it here, since ticket 45 just says "admin edit functionality may already be implemented so could just need testing" and that way I dont have to wait for another pull request to be approved.